Source/ISHRemote/Trisoft.ISHRemote/Cmdlets/BackgroundTask/AddIshBackgroundTask.Tests.ps1

```diff
@@ -0,0 +1,85 @@
+﻿Write-Host ("`r`nLoading ISHRemote.PesterSetup.ps1 for MyCommand[" + $MyInvocation.MyCommand + "]...")
+. (Join-Path (Split-Path -Parent $MyInvocation.MyCommand.Path) "..\..\ISHRemote.PesterSetup.ps1")
+$cmdletName = "Add-IshBackgroundTask"
+try {
+
+Describe “Add-IshBackgroundTask" -Tags "Create" {
+	Write-Host "Initializing Test Data and Variables"
+	$eventType = "PUSHTRANSLATIONS"
+	$ishFolderTestRootOriginal = Get-IshFolder -IShSession $ishSession -FolderPath $folderTestRootPath
+	$folderIdTestRootOriginal = $ishFolderTestRootOriginal.IshFolderRef
+	$folderTypeTestRootOriginal = $ishFolderTestRootOriginal.IshFolderType
+	$ownedByTestRootOriginal = $ishFolderTestRootOriginal.fusergroup_none_element
+	$readAccessTestRootOriginal = $ishFolderTestRootOriginal.readaccess_none_element
+	
+    Write-Debug("folderIdTestRootOriginal[" +  $ishFolderTestRootOriginal.IshFolderRef + "] folderTypeTestRootOriginal[" + $folderTypeTestRootOriginal + "]")
+	$global:ishBackgroundTaskCmdlet = Add-IshFolder -IShSession $ishSession -ParentFolderId $folderIdTestRootOriginal -FolderType $folderTypeTestRootOriginal -FolderName $cmdletName -OwnedBy $ownedByTestRootOriginal -ReadAccess $readAccessTestRootOriginal
+	$ishFolderTopic = Add-IshFolder -IshSession $ishSession -ParentFolderId($global:ishBackgroundTaskCmdlet.IshFolderRef) -FolderType ISHModule -FolderName "Topic" -OwnedBy $ownedByTestRootOriginal -ReadAccess $readAccessTestRootOriginal
+
+	$ishTopicMetadata = Set-IshMetadataField -IshSession $ishSession -Name "FTITLE" -Level Logical -Value "Topic $timestamp" |
+						    Set-IshMetadataField -IshSession $ishSession -Name "FAUTHOR" -Level Lng -ValueType Element -Value $ishUserAuthor |
+						    Set-IshMetadataField -IshSession $ishSession -Name "FSTATUS" -Level Lng -ValueType Element -Value $ishStatusDraft
+    $ishObjectTopic1 = Add-IshDocumentObj -IshSession $ishSession -FolderId $ishFolderTopic.IshFolderRef -IshType ISHModule -Lng $ishLng -Metadata $ishTopicMetadata -FileContent $ditaTopicFileContent
+    $ishObjectTopic1 = Add-IshDocumentObj -IshSession $ishSession -FolderId $ishFolderTopic.IshFolderRef -IshType ISHModule -Lng $ishLng -Metadata $ishTopicMetadata -FileContent $ditaTopicFileContent
+
+	$ishObjects = $ishFolderTopic | Get-IshFolderContent -IshSession $ishSession
+	Context "Add-IshBackgroundTask IshObjects passed via pipeline" {
+		$ishBackgroundTaskIshObjectsPipeline = $ishObjects | Add-IshBackgroundTask -IshSession $ishSession -EventType $eventType
+		It "Verify object properties returned by Add-IshBackgroundTask" {
+			$ishBackgroundTaskIshObjectsPipeline.Count | Should BeExactly 1
+			$ishBackgroundTaskIshObjectsPipeline.status_task_element | Should BeExactly "VBACKGROUNDTASKSTATUSPENDING"
```

Comment:
Don't check on the status of the background task unless you stop the background task service this will not be correct

Reply:
Done